docs/WritingTests.asciidoc

```diff
@@ -235,7 +239,7 @@ clone_job.pl --from https://openqa.opensuse.org  --host localhost 24 SKIPTO=cons
 * Use qemu-img snapshot -l something.img to find out what snapshots are in the image. Snapshots are named
 `"test module category"-"test module name"` (e.g. `installation-start_install`)
 
-==== Storing only the last sucessfull snapshot
+==== Storing only the last sucessful snapshot
 
 * Run the worker with +--no-cleanup parameter+. This will preserve the hard disks after test runs.
 * Set +TESTDEBUG=1+ on a job. This will make openQA save a snapshot after each successful test module run. Snapshots are overwritten.
```

Comment:
i would mention that they are being saved as "lastgood" since in the previous section, format of snapshots are being mentioned when `MAKETESTSNAPSHOTS=1`